packages/utils/src/fs.ts

```diff
@@ -235,3 +214,13 @@ export class DiskFS implements FS {
     return realpathSync(this.getPath(path));
   }
 }
+
+export function readFileAndThrowOnBOM(fileName: string, fs: FS): string {
```

Comment:
This function was pulled out of a deleted file somewhere, but quite frankly I'm not sure if I should keep this around. Pretty wild that it globally installs a random CLI. It's only used for file hashing so it's a little hard for me to justify it...

Reply:
I'm dumb. This is an error message. Nevermind.